### PR TITLE
Zephyr - SOF: Patch set

### PR DESCRIPTION
**Pull request by finikorg**
Add building base_module and rimage tool to Zephyr.

### CODE REVIEW

File: zephyr/include/sof/panic.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ */
+
+#ifndef __INCLUDE_SOF_IPC_PANIC__
```
Comment (jajanusz) [inline zephyr/include/sof/panic.h]: Defguards should be related to paths, there is no 'IPC' in path.

File: zephyr/CMakeLists.txt [hunk 2]
```diff
@@ -110,4 +110,12 @@ zephyr_library_sources_ifdef(CONFIG_SOF_USE_DMIC
 
 zephyr_library_link_libraries(SOF)
 target_link_libraries(SOF INTERFACE zephyr_interface)
+
+add_library(sof_base_module
+	../src/platform/apollolake/base_module.c
+	)
```
Comment (jajanusz) [inline zephyr/CMakeLists.txt]: No need to ident ending parenthesis.

**Comment by jajanusz** [comment]
@finikorg  BTW I think you can push directly to your branch, PRs are mandatory only for master branch.

**Comment by paulstelian97** [comment]
> @finikorg BTW I think you can push directly to your branch, PRs are mandatory only for master branch.

Of course he should only do that if he doesn't want review. I guess it's his choice, if there are many issues whenever he will want to merge with master it may be better to figure them out as he goes rather than at that point.

**Comment by tlauda** [comment]
@finikorg Do you want me to merge it?

**Comment by finikorg** [comment]
Closing outdated PR

**Pull request closed without merging**